extractor/filesystem/secrets/convert/convert.go

```diff
@@ -45,6 +49,8 @@ type detectorWrapper struct {
 	version       int
 }
 
+func (d *detectorWrapper) IsPlaceholder() bool { return true }
```

Comment:
technically the FromVelesDetectorWithRequire extractors are also placeholders since they're added to the Veles engine. How about we invert this and call it "KeepExtractor" ?

Reply:
Also add a comment about what this does (keeps the standalone copy of the extractor for the use case where we want to look at additional files)